lightning/src/onion_message/messenger.rs

```diff
@@ -1081,6 +1112,13 @@ where
 						e.get_mut().enqueue_message(onion_message);
 						log_trace!(logger, "Forwarding an onion message to peer {}", next_node_id);
 					},
+					_ if self.intercept_messages_for_offline_peers => {
+						self.pending_events.lock().unwrap().push(
+							Event::OnionMessageIntercepted {
+								peer_node_id: next_node_id, message: onion_message
+							}
+						);
+					},
```

Comment:
Could we `log_trace` here like the other arms?

Reply:
I omitted it because of our past discussions of avoiding adding logs that would allow someone to adversarially blow up our logs on disk (there are some pre-existing ones in this method, ofc). So I'm not sure it's a good idea?

Reply:
Hmm... I thought that was only applicable for higher-level logging like `log_error`. Will let @TheBlueMatt provide guidance.

Reply:
Sure, happy to add it in follow-up too, no reason to hold the PR up.